docusaurus/docs/e2e-test-a-plugin/get-started.md

```diff
@@ -167,7 +176,7 @@ queryString="current-package-manager"
 
 ### Step 6: Run tests in CI
 
-We recommend updating your CI workflow to include the running of end-to-end tests to continuously check for breakages. The following workflow can be used to run end-to-end tests against a range of Grafana versions for every PR in your Github repository. Note that this is a generic example based on a backend plugin. You may want to alter or remove a few of the steps in the `playwright-tests` job before using it in your plugin.
```

Comment:
Why remove the recommendation?

Reply:
I added the `no-hasGithubWorkflows` args, so now there's no workflow to update. But I can rephrase this sentence instead.